DQM/Integration/python/clients/pixel_dqm_sourceclient-live_cfg.py

```diff
@@ -73,6 +73,7 @@
 # Real data raw to digi
 process.load("EventFilter.SiPixelRawToDigi.SiPixelRawToDigi_cfi")
 process.siPixelDigis.IncludeErrors = True
+process.siPixelDigis.UsePhase1 = True
```

Comment:
hi @fioriNTU - it seems to run ok for me (not convinced I processed events but my job run over a bunch of files from run 295613 eating cpu..)